api/SpeechRecognitionEvent.json

```diff
@@ -53,6 +53,59 @@
           "deprecated": false
         }
       },
+      "SpeechRecognitionEvent": {
+        "__compat": {
+          "description": "<code>SpeechRecognitionEvent()</code> constructor",
+          "support": {
+            "chrome": {
+              "prefix": "webkit",
+              "version_added": "33"
+            },
+            "chrome_android": {
+              "prefix": "webkit",
+              "version_added": true
+            },
+            "edge": {
+              "prefix": "webkit",
+              "version_added": "≤79"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": "14.1"
+            },
+            "safari_ios": {
+              "version_added": "14.5"
+            },
+            "samsunginternet_android": {
+              "prefix": "webkit",
+              "version_added": true
+            },
+            "webview_android": {
+              "prefix": "webkit",
+              "version_added": "≤37"
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
```

Comment:
It's https://wicg.github.io/speech-api/#dom-speechrecognitionevent-speechrecognitionevent, can you link that and set `standard_track` to true?